radio/src/targets/horus/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ set(PPM_LIMITS_SYMETRICAL YES)
 set(USB_SERIAL ON CACHE BOOL "Enable USB serial (CDC)")
 set(ROTARY_ENCODER YES)
 set(HARDWARE_EXTERNAL_MODULE YES)
+set(FLEXSW "0" CACHE STRING "Max allowed user defined flex used as switches")
```

Comment:
I believe we should be able to define this option in `hal/CMakeLists.txt` instead of repeating it for each target.